### PR TITLE
Bug/shop-tab // Fix shop tab bugs

### PR DESCRIPTION
**Pull request by Aslanyan99n**
- Fixed the issue with images appearing after searching for an item.
- Disabled card interaction after clicking the favorite button.
- Fix shop icons on listing.
- Icon on shop can be not round.

### CODE REVIEW

File: Birthday/Birthday/Modules/Home/HomePage/ViewModel/HomeViewModel.swift [hunk 1]
```diff
@@ -22,9 +22,7 @@ final class HomeViewModel: HomeViewModeling {
   }
   
   func getBirthDays() {
-    withAnimation {
-      isLoading = true
-    }
+    isLoading = true
```
Comment (AkselAvetisyan) [inline Birthday/Birthday/Modules/Home/HomePage/ViewModel/HomeViewModel.swift]: Why did you remove this animation?

Comment (Aslanyan99n) [reply Birthday/Birthday/Modules/Home/HomePage/ViewModel/HomeViewModel.swift]: When we navigate from signIn to Home Screen it seems buggy.

Comment (Aslanyan99n) [reply Birthday/Birthday/Modules/Home/HomePage/ViewModel/HomeViewModel.swift]: Fixed

File: Birthday/Birthday/Modules/Home/Birthday Details/View/BirthdayDetailsScreen.swift [hunk 2]
```diff
@@ -171,6 +125,37 @@ extension BirthdayDetailsScreen {
     }
   }
   
+  @ViewBuilder
+    private var pickerImage: some View {
```
Comment (AkselAvetisyan) [inline Birthday/Birthday/Modules/Home/Birthday Details/View/BirthdayDetailsScreen.swift]: Fix the indentation here

Comment (Aslanyan99n) [reply Birthday/Birthday/Modules/Home/Birthday Details/View/BirthdayDetailsScreen.swift]: Fixed

**Review by AkselAvetisyan — APPROVED** [review]
